plugins/mcreator-localization/lang/texts.properties

```diff
@@ -171,6 +171,14 @@ trigger.entity_pickupxp=Entity picks up experience
 trigger.player_useitem_start=Player uses item
 trigger.player_useitem_stop=Player stops using item
 trigger.player_useitem_finish=Player finishes using item
+trigger.entity_sets_attack_target=Entity sets its attack target
+trigger.entity_grief=Entity about to grief
+trigger.player_xp_change=Player xp amount changes
+trigger.player_xp_level_change=Player xp level changes
+trigger.player_can_critical_hit=Player can deal critical hit

```

Comment:
Rename this to `Player deals critical hit`, right now this sounds more like a condition than a trigger. Also, change machine name.